cts/valgrind-pcmk.suppressions

```diff
@@ -272,24 +306,58 @@
 }
 
 {
-   glib - why do you hate me?
+   glib - hashtable new calloc
    Memcheck:Leak
+   match-leak-kinds: reachable
    fun:calloc
    fun:g_malloc0
-   obj:*/libgobject-*
-   obj:*/libgobject-*
-   fun:call_init.part.0
+   ...
+   fun:g_hash_table_new_full
+   ...
+   obj:*/libglib-*
+   ...
+   fun:call_init
    fun:_dl_init
 }
 
 {
-   dear glib - you suck at memory management
+   glib - hashtable new malloc
    Memcheck:Leak
-   fun:calloc
-   fun:g_malloc0
-   obj:*/libgobject-*
-   obj:*/libgobject-*
-   obj:*/libgobject-*
-   fun:call_init.part.0
+   match-leak-kinds: reachable
+   fun:malloc
+   ...
+   fun:g_hash_table_new_full
+   ...
+   obj:*/libglib-*
+   ...
+   fun:call_init
+   fun:_dl_init
+}
+
+{
+   glib - hashtable new malloc 2
+   Memcheck:Leak
+   match-leak-kinds: reachable
+   fun:malloc
+   fun:g_malloc
+   ...
+   obj:*/libglib-*
+   ...
+   fun:call_init
+   fun:_dl_init
+}
+
+{
+   glib - hashtable new realloc
+   Memcheck:Leak
+   match-leak-kinds: reachable
```

Comment:
are we ignoring things glib leaves reachable, or things we leave reachable?

Reply:
I was hoping to ignore things that glib allocates but does not clean up.  Anything we allocate but do not clean up we should warn about.